content/docs/1.0.0/tutorial/deploy-your-microservice.md

```diff
@@ -86,34 +85,114 @@ With the microservice UUID from the last step, let's change our routes so that o
 
 First, let's remove the old route from the Sensors to the REST API. We need to retrieve the microservice UUIDs for Sensors and the REST API:
 
-```sh
-iofog-controller microservice list
+```bash
+docker exec -ti iofog-controller iofog-controller microservice list
 ```
 
-After finding those two UUIDs in the list, provide the Sensors UUID and API UUID separated by a semicolon to `microservice route-remove --route <source_uuid>:<dest_uuid>`:
+Example output can look like this. Note that many attributes are not shown in this output, but we can clearly see all the UUIDs and routes.
```

Comment:
**An** example output can look like **the following**. Note that **although** many attributes are not shown in this output, ~but~ we can clearly...